source/_patterns/01-molecules/26-top-offer-list/top-offer-hotel-into.mustache

```diff
@@ -0,0 +1,23 @@
+<div class="top-offer-hotel-info">
+
+  <aside class="top-offer-recommendation">
+    <div>
+      {{> molecules-review-rate-green}}
+    </div>
+    <div class="review-rate-info">
+      {{> atoms-simple-span-text(spanContent: 'empfehlen dieses Hotel weiter')}}
+      <span>
+        (75 Bewertungen)
+      </span>
+    </div>
+  </aside>
+
+  <header class="top-offer-title">
+    {{> atoms-h3(h3Content:'An insanely loooohoooooooong Hotel name ')}}&nbsp;
+    {{> molecules-hotel-category-images}}
+  </header>
+
+
+  {{> atoms-simple-p-text(pContent: 'Bahia Feliz - Tarajalillo in San Agustin, Gran Canaria')  }}
+  {{> atoms-Anchor(AnchorContent: 'Hoteldetails') }}
```

Comment:
Is this supposed to be capitalised? "atoms-Anchor"

Reply:
Oh, yes. Thanks.